src/components/screens/wallet/send/form/formBtc.js

```diff
@@ -114,14 +120,14 @@ export default class FormBtc extends React.Component {
   }
 
   // TODO move dynamic fee calculation and presentation to a separate component
```

Comment:
💅🏻based on our conversation this comment can be remove as the function will remain in this file